cli/azd/cmd/infra_create.go

```diff
@@ -122,7 +126,9 @@ func (i *infraCreateAction) Run(ctx context.Context) (*actions.ActionResult, err
 		return nil, fmt.Errorf("planning deployment: %w", err)
 	}
 
-	provisioningScope := infra.NewSubscriptionScope(ctx, env.GetLocation(), env.GetSubscriptionId(), env.GetEnvName())
+	provisioningScope := infra.NewSubscriptionScope(
```

Comment:
Let's pull out `ctx` from the subscription / resourceGroup scope constructors

Reply:
Good call.